client/src/Main.purs

```diff
@@ -140,7 +139,11 @@ compile = do
         liftEffect cleanUpMarkers
 
         case res of
-          Right (CompileSuccess (SuccessResult { js, warnings })) -> do
+          Left err -> liftEffect do
```

Comment:
This implementation has barely changed; I've just moved it from the end of the case statement to the top of the case statement.